libminifi/src/controllers/SSLContextService.cpp

```diff
@@ -269,10 +505,8 @@ void SSLContextService::onEnable() {
     }
     passphrase_file.close();
   }
-  // load CA certificates
-  if (!getProperty(caCert.getName(), ca_certificate_)) {
-    logger_->log_error("Can not load CA certificate.");
```

Comment:
is the log no longer necessary?

Reply:
It isn't an error any longer, because now there is a valid use case when the server certificate name is blank and `nifi.security.use.system.cert.store=true`.  In that case, we will try to find the server certificate in the system store.